aws.go

```diff
@@ -76,6 +76,9 @@ func awsGetLaunchTemplate(svc ec2iface.EC2API, input *ec2.DescribeLaunchTemplate
 	return templatesOutput.LaunchTemplates[0], nil
 }
 func awsGetHostnames(svc ec2iface.EC2API, ids []string) ([]string, error) {
+	if len(ids) == 0 {
+		return []string{}, nil
+	}
```

Comment:
It obviously doesn't hurt, but why is it needed? Wouldn't the next lines get it? Or is it to avoid bothering making the API call in the first place?

Reply:
You're right - it's to avoid making the API call in the first place.

The API call will in fact return a 400s because it expects at least one instance id to be passed as parameter, meaning that no matter what, the following line would never be reached:

```golang
if len(templatesOutput.LaunchTemplates) < 1 {
	return nil, nil
}
```

With this change, it will be reached, as intended.